easybuild/easyblocks/p/python_meep.py

```diff
@@ -56,6 +56,9 @@ def configure_step(self):
         for dep in deps:
             if not get_software_root(dep):
                 raise EasyBuildError("Module for %s not loaded.", dep)
+        # prepare easyblock by determining Python site lib dir
+        if self.pylibdir == UNKNOWN:
+            self.pylibdir = det_pylibdir()
```

Comment:
actually, a better fix would be to simply add the missing super call:

```python
super(EB_python_minus_meep, self).configure_step()
```

That will take care of this too, and more (debug logging w.r.t. Python version being used, for example)